addons/project/views/project_views.xml

```diff
@@ -1105,8 +1105,11 @@
                                 <field name="priority" widget="priority_switch" class="me-3"/>
                                 <field name="name" class="o_task_name text-truncate w-100 w-md-75 pe-2" placeholder="Task Title..."/>
                             </div>
-                            <div class="d-flex justify-content-end o_state_container">
-                                <field name="state" widget="project_task_state_selection" class="o_task_state_widget"/>
+                            <div class="d-flex justify-content-end o_state_container" attrs="{'invisible': [('active', '=', False)]}">
```

Comment:
No need to add a space for task-<task_id> 🙂 

```diff
- task - 3374980
+ task-3374980
```

I will do the change in your commit. 🙂 

Can you please apply that in your next PRs? Thanks in advance!

Reply:
sure, I will do that.